node-graphql/server.mongo.js

```diff
@@ -20,7 +20,7 @@ let AWS = require('aws-sdk');
 // });
 
 // We connect mongoose to our local mongodb database
-const connection = mongoose.connect('mongodb://dashboard-mongo:27017/dashboard', {
+const connection = mongoose.connect('mongodb://dashboard-mongo:27030/dashboard', {
```

Comment:
If you are changing the port in this file, you probably also need to modify the prod docker-compose port as well.